core/assets/bundles/bundle_vi.properties

```diff
@@ -584,22 +584,22 @@ planet.serpulo.name = Serpulo
 planet.sun.name = Mặt trời
 
 #Why we should translate this ??
-sector.impact0078.name = Impact 0078
-sector.groundZero.name = Ground Zero
-sector.craters.name = The Craters
-sector.frozenForest.name = Frozen Forest
-sector.ruinousShores.name = Ruinous Shores
-sector.stainedMountains.name = Stained Mountains
-sector.desolateRift.name = Desolate Rift
-sector.nuclearComplex.name = Nuclear Production Complex
-sector.overgrowth.name = Overgrowth
-sector.tarFields.name = Tar Fields
-sector.saltFlats.name = Salt Flats
-sector.fungalPass.name = Fungal Pass
-sector.biomassFacility.name = Biomass Synthesis Facility
-sector.windsweptIslands.name = Windswept Islands
-sector.extractionOutpost.name = Extraction Outpost
-sector.planetaryTerminal.name = Planetary Launch Terminal
+sector.impact0078.name = Va Chạm số 0078
+sector.groundZero.name = Vị Trí 0
+sector.craters.name = Vực Sâu
+sector.frozenForest.name = Khu Rừng Đóng Băng
+sector.ruinousShores.name = Bờ Biển Tàn Phá
+sector.stainedMountains.name = Dãy Núi Ô Uế
+sector.desolateRift.name = Thung Lũng Tuyệt Vọng
+sector.nuclearComplex.name = Khu Sản Xuất Nguyên Tử
+sector.overgrowth.name = Khu Rừng Bào Tử
+sector.tarFields.name = Hoang Mạc Dầu Hắc
+sector.saltFlats.name = Hoang Mạc Muối
+sector.fungalPass.name = Con Đèo Bào Tử
+sector.biomassFacility.name = Khu Bào Chế Bào Tử
+sector.windsweptIslands.name = Những Hòn Đảo Giông Gió
+sector.extractionOutpost.name = Vị Trí Khởi Hành
+sector.planetaryTerminal.name = Khu Du Hành Hành Tinh
```

Comment:
Planetary Travel Area? why

Reply:
it is actually corect
google translate doesnt know what i should be saying

Reply:
it's same as PLT , but ggogle wont help but butcher the meaning